app/server/routes/v1/user.js

```diff
@@ -139,6 +140,9 @@ router.patch("/", async (req, res) => {
     let query = { $set: {} };
 
     Object.keys(req.body).forEach((key) => {
+      if (key === "name")
+        return res.status(403).send("Not allowed to edit name.");
+
       // if the field in req.body exists, update/set it
       if (user[key] && user[key] !== req.body[key]) {
         query.$set[key] = req.body[key];
```

Comment:
Regarding our other conversation, if a name key is not included in the request, will the user still have a name key after this? 
It seems like it starts with an empty object and only sets/updates keys that are passed. 
If a name key is passed, it will fail (good), but if not then it doesn't seem like the new object will have a name key anymore.

Reply:
It only updates the keys that are passed in the request body, except if you try to set the name field it will return a response saying you can't edit that field. The user document will still include a name.